### ISSUE TITLE
Disable overscroll plugin

### ISSUE THREAD

**Issue by emko4** [idiotWu/react-smooth-scrollbar#19]
HI, I don't know if its bug of OverscrollPlugin, smooth-scrollbar or react-smooth-scrollbar, but when I try disable OverscrollPlugin by 
```
plugins: { overscroll: false },
```
scrollbar events/plugins stop working, but there isn't error in console.

I have workaround,
```
plugins={{ overscroll: { effect: 'glow', glowColor: 'transparent' } }}
```
but I don't like it :)

**Comment by dolphin-wood** [comment]
You can disable the overscroll plugin by:

1. exclude `OverscrollPlugin` when call `Scrollbar.use()` (globally disabled)
2. pass `overscroll: false` when **initializing** scrollbars. Set `overscroll` to `false` lately will have no effect.
3. set `overscroll.effect` to `null`

**Comment by emko4** [comment]
1. I need disable  `OverscrollPlugin` only in few cases

2. I have more plugins e.g. `Scrollbar.use(MyPlugin1, MyPlugin2, OverscrollPlugin)` and when I use `overscroll: false`, all of them stop working

3.  Worked before few weeks, but same behavior like **2.** now

**Comment by dolphin-wood** [comment]
I can't reproduce this issue with smooth-scrollbar@8.2.0. Which version are you using?

**Comment by emko4** [comment]
I'm using `smooth-scrollbar@8.2.0` and `react-smooth-scrollbar@8.0.6`.

I'm using two plugins [InvertDeltaPlugin](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/plugin.md#example-invert-delta) and `OverscrollPlugin`

Plugin use: `Scrollbar.use(InvertDeltaPlugin, OverscrollPlugin)`

Scrollbar definition:
```
<Scrollbar
    onScroll={this.onScrollHandler}
    plugins={{  invertDelta: { enable: true }, overscroll: false }}
>
    {content}
</Scrollbar>
```

When `overscroll` is `false`, invertDelta stop working too (`transformDelta` isn't called).
When `overscroll` is `{ effect: 'bounce' }`, both plugins work.

But maybe it's something else in my code :(

**Comment by dolphin-wood** [comment]
That is weird. I tried the same code but it worked as expected.

**Comment by emko4** [comment]
Ok, I will try debug more :) thanks

**Comment by emko4** [comment]
Well, first I'm sorry, because issue isn't in `OverscrollPlugin`. After some tests, I found out, that even [NoopPlugin](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/plugin.md#plugin-order) doesn't work. So I tried use `NoopPlugin` alone (without `OverscrollPlugin`) and still nothing.

Long story short, I have content scrollable only horizontally (content has same height like `Scrollbar`) and problem is in `shoulePropagateMomentum` (there is a typo 😄 ) for wheel event.

I think that  `shoulePropagateMomentum` should return true.

Something like small stackTrace:
`shoulePropagateMomentum` (node_modules\smooth-scrollbar\utils\shoule-propagate-momentum.js)
`wheelHandler` (node_modules\smooth-scrollbar\events\wheel.js)

If it isn't bug, but feature, I'm ok with it 😄and thanks for the best scrollbar library!
